ocaml/xapi/xapi_vm_helpers.ml

```diff
@@ -1579,3 +1579,11 @@ let ensure_device_model_profile_present ~__context ~domain_type ~is_a_template
   else
     (* only add device-model to an HVM VM platform if it is not already there *)
     default :: platform
+
+let remove_pending_guidance ~__context ~self ~value =
```

Comment:
Is this more or less the same code as above? Why have the same function twice?

Reply:
I think one is for VM, and another is for host.